snapcraft/internal/cache/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from snapcraft.internal.cache._cache import rewrite_snap_filename_with_revision  # noqa
+from snapcraft.internal.cache._cache import SnapcraftCache  # noqa
+from snapcraft.internal.cache._cache import SnapcraftProjectCache  # noqa
+from snapcraft.internal.cache._cache import SnapCache  # noqa
```

Comment:
It seems for now only this guy needs to be exported. If testing is the only reason, using the ugly names with _ seems better.